src/content/en/fundamentals/getting-started/service-worker/rough-edges-and-gotchas.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: shared/narrow
+title: "Rough Edges and Gotchas"
+description: "This stuff is really new. Here's a collection of issues that get in the way. "
+published_on: 2014-12-01
+updated_on: 2016-01-19
+translation_priority: 0
+order: 9
+authors:
+  - mattgaunt
+---
+
+{% include shared/toc.liquid %}
+
+This stuff is really new. Here's a collection of issues that get in the way.
+Hopefully this section can be deleted soon, but for now these are worth being
+mindful of.
+
+## If Installation Fails, We're Not so Good at Telling You About It
+
+If a worker registers, but then doesn't appear in `chrome://inspect/#service-
+workers` or `chrome://serviceworker-internals`, it's likely failed to
+install due to an error being thrown, or a rejected promise being passed to
+`event.waitUntil()`.
+
+To work around this, go to `chrome://serviceworker-internals` and check "Open
+DevTools window and pause JavaScript execution on Service Worker startup for
+debugging.", and put a debugger statement at the start of your install event.
+(This option is named differently in versions of Chrome earlier than 47.)
+This, along with  "[Pause on uncaught exceptions](https://developer.chrome.com/devtools/docs/javascript-debugging
```

Comment:
Writer thing. Control names should be handled in a consistent manner even if it seems redundant in individual cases. I've been trying to use quotes on control names throughout the site.